### PR TITLE
Tech_forum: demo of git, gitHub, sonarQube and GitFlow.

### PR DESCRIPTION
**Pull request by hammerdj1**

### CODE REVIEW

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 1]
```diff
@@ -50,6 +52,9 @@ public ContactsExampleRoute() {
     public void configure() throws Exception {
         // you can configure the route rule with Java DSL here
 
+        String unusedVar = "unused";
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Remove this unused "unusedVar" local variable. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "unusedVar". [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1854)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 6 issues
* MAJOR 4 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 2]
```diff
@@ -50,6 +52,16 @@ public ContactsExampleRoute() {
...
+        String WRONG_CASE = "will it find this?";
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Remove this unused "WRONG_CASE" local variable. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "WRONG_CASE". [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1854)
MINOR Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS00117)

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 3]
```diff
@@ -50,6 +52,16 @@ public ContactsExampleRoute() {
...
+
+        try {
+            System.out.println("ContactsExampleRoute.configure");
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Replace this usage of System.out or System.err by a logger. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS106)

File: src/main/java/com/inmarsat/demo/ContactsExampleRoute.java [hunk 4]
```diff
@@ -50,6 +52,16 @@ public ContactsExampleRoute() {
...
+        } catch (Exception e) {
```
Comment (cloudbotinmarsat) [inline src/main/java/com/inmarsat/demo/ContactsExampleRoute.java]: MAJOR Either log or rethrow this exception. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1166)
MAJOR Either remove or fill this block of code. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=squid%3AS00108)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 10 issues
* MAJOR 8 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by hammerdj1** [comment]
test this please

**Comment by hammerdj1** [comment]
Rubbish!

**Pull request closed without merging**